src/cli/core/FileContentGenerator.ts

```diff
@@ -81,14 +81,14 @@ export class FileContentGenerator {
     return (
       'export type TTailwindString = "TAILWIND_STRING"\n' +
       '\n' +
-      'export type TKey = TClasses | TTailwindString\n' +
+      'export type TKey = TClasses | TTailwindStringIMPORTED_T_CUSTOM_CLASSES_KEY\n' +
```

Comment:
I split the predefined placeholder string `IMPORTED_T_CUSTOM_CLASSES` into `IMPORTED_T_CUSTOM_CLASSES_KEY` and `IMPORTED_T_CUSTOM_CLASSES_ARG`. The former contains a leading space whereas the latter does not.